paper/paper.bib

```diff
@@ -105,15 +105,16 @@ @article{nageli2020towards
   doi = {10.1016/j.enbuild.2020.109763}
 }
 
-@article{anatolitis2017putting,
```

Comment:
Agent-based energy markets are quite hot right now, so it might also be useful to keep this one.

@jackiekazil I think you can just add the papers you want without removing any.